### PR TITLE
build: Release v39.0

### PR DESCRIPTION
**Pull request by rbradford**
Signed-off-by: Rob Bradford <rbradford@rivosinc.com>

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -327,6 +334,77 @@
...
+* Fix documentation to indicate only stream mode is supported by `virtio-vsock`
+  (#6306)
+* Fix `virtio-fs` tag validation (#6358, #6359)
+* Add missing `pvpanic` device to OpenAPI metadata (#6372)
+* Fixes for nested virtualization with VFIO devices (#6110, #6298, #6297,
+  #6319)
+* Fixes for backing file for `virtio-mem` regions with snapshot/restore (#6337,
+  #6338)
+* Explicitly mark FDs used for network devices as invalid across
+  snapshot/restore (#6332, 6286)
```
Comment (likebreath) [inline release-notes.md:380]: Typo: `\s6286\#6286`

File: release-notes.md [hunk 2]
```diff
@@ -327,6 +334,77 @@
...
+* Correctly cleanup `sigwinch_listener` process (#6208)
+* Graceful shutdown of HTTP API thread (#6248, #6247)
+* Fix `queue_affinity` option in OpenAPI metadata (#6268)
+* Fix documentation to indicate only stream mode is supported by `virtio-vsock`
+  (#6306)
+* Fix `virtio-fs` tag validation (#6358, #6359)
+* Add missing `pvpanic` device to OpenAPI metadata (#6372)
+* Fixes for nested virtualization with VFIO devices (#6110, #6298, #6297,
+  #6319)
+* Fixes for backing file for `virtio-mem` regions with snapshot/restore (#6337,
```
Comment (likebreath) [inline release-notes.md:377]: Nit: better to stay with imperative tone, say `\sFixes for\Fix`

File: release-notes.md [hunk 3]
```diff
@@ -327,6 +334,77 @@
...
+  #6338)
+* Explicitly mark FDs used for network devices as invalid across
+  snapshot/restore (#6332, 6286)
+* Improved `event-monitor` events around reboot (#6277, #6274)
```
Comment (likebreath) [inline release-notes.md:381]: Ditto: `\sImproved\Improve`

**Review by likebreath — APPROVED** [review]
@rbradford Thank you. Few nits below.

File: release-notes.md (hunk 2 above)
Comment (rbradford) [reply release-notes.md:377]: Ah, one of them was wrong - but the nested virtualization one is plural.

**Pull request merged**